fcrepo-http-api/src/main/resources/views/common.js

```diff
@@ -94,7 +94,15 @@
       const url = this.href;
       http('HEAD', url, function(res) {
         if (res.status >= 400 || res.getResponseHeader('Link') == null) {
-          location.href = url;
+          var newLocation = url;
```

Comment:
I shared the same thoughts when reviewing this PR... but was also unable to determine a more reliable, LDP-based solution.
I would suggest add to the inline comments:
```
// WARNING: Fragile code relying on magic suffix '/fcr:metadata' and absence of 'Link' header on external resource.
```